docs/learning/tutorials/development-prerequisites.md

```diff
@@ -2,7 +2,7 @@
 
 Writing an iTwin.js application requires the following software:
 
-- [Node.js](https://nodejs.org) (latest 10.x or 12.x LTS version)
+- [Node.js](https://nodejs.org) (14.x LTS version)

```

Comment:
Should we continue to list 12.x LTS as a supported version here

Reply:
Nah I don't think it's required, this is mostly our suggestions for building an app and pretty much everyone should be using 14.x.